src/Sylius/Component/Order/Model/OrderInterface.php

```diff
@@ -28,119 +28,119 @@
     public const STATE_FULFILLED = 'fulfilled';
 
     /**
-     * @return \DateTimeInterface
+     * @return \DateTimeInterface|null
      */
-    public function getCheckoutCompletedAt();
+    public function getCheckoutCompletedAt(): ?\DateTimeInterface;
 
     /**
-     * @param null|\DateTimeInterface $checkoutCompletedAt
+     * @param \DateTimeInterface|null $checkoutCompletedAt
      */
-    public function setCheckoutCompletedAt(\DateTimeInterface $checkoutCompletedAt = null);
+    public function setCheckoutCompletedAt(?\DateTimeInterface $checkoutCompletedAt): void;
```

Comment:
And this and everything else where we remove the default value from public method.